renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "automerge": true,
+      "matchUpdateTypes": [
+        "patch",
+        "pin",
+        "digest"
+      ]
+    },
+    {
+      "groupName": "terraform",
+      "managers": [
+        "terraform"
```

Comment:
This will group only terraform minor updates into a single PR. We can probably extend that to pre-commit, and golang updates as well.

Reply:
Do you think all three should be in one PR or do we want one PR for each package type?

Reply:
I think reducing the volume of PRs is the main goal and am willing to trade off the occasional PR that will include a list of unrelated dependency updates.